packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -114,11 +166,95 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
                         </WalletText>
                     </div>
                     <div className='wallets-mt5-trade-screen__details-description__balance'>
-                        {shouldShowAccountBalance && <WalletText weight='bold'>{details?.display_balance}</WalletText>}
+                        {platform !== ctraderPlatform && shouldShowAccountBalance ? (
+                            <WalletText weight='bold'>{details?.display_balance}</WalletText>
+                        ) : (
+                            <div className='wallets-mt5-trade-screen__details-description__balance-total'>
+                                <WalletText as='p' lineHeight='3xs' size={isDesktop ? 'sm' : 'md'}>
+                                    Total balance:
+                                </WalletText>
+                                <WalletText weight='bold'>
+                                    {ctraderTotalBalance} {activeWalletData?.currency}
+                                </WalletText>
+                            </div>
+                        )}
                         {migrationMessage}
                     </div>
                 </div>
 
+                {platform === ctraderPlatform && (
+                    <>
+                        <ul className='wallets-mt5-trade-screen__ctrader-account-list'>
+                            {sortedCtraderAccountList?.map(account => (
+                                <li className='wallets-mt5-trade-screen__ctrader-account-list-item' key={account.login}>
+                                    <WalletText size='sm'>{account.login}</WalletText>
+                                    <WalletText size='sm' weight='bold'>
+                                        {account.formatted_balance}
+                                    </WalletText>
+                                </li>
+                            ))}
+                        </ul>
+                        {availableCtraderAccounts?.[0].available_count !== undefined &&
+                            availableCtraderAccounts[0].available_count > 0 && (
+                                <button
+                                    className='wallets-mt5-trade-screen__ctrader-get-more-button'
+                                    onClick={onClickCtraderGetMoreButton}
+                                    type='button'
+                                >
+                                    <span className='wallets-mt5-trade-screen__ctrader-get-more-button-icon'>
+                                        <LabelPairedPlusMdFillIcon fill='#000000' />
+                                    </span>
+                                    <WalletText size='xs'>Get another cTrader cTrader account</WalletText>
+                                </button>
```

Comment:
can we use WalletButton or icon Button instead?

Reply:
I tried. But design is totally different